shared/src/libs/helpers/debounce-function/debounce-function.helper.ts

```diff
@@ -0,0 +1,14 @@
+import debounce from 'debounce';
```

Comment:
let's rename import to libraryDebounce and rename our function to debounce 
```suggestion
import libraryDebounce from 'debounce';
```

Reply:
Done